### DISCUSSION TITLE
Simultainious convert to use all availible cores?

### DISCUSSION THREAD

**Discussion by pwnusmaximus** [openaudible/openaudible#859] [Ideas]
Dear openaudible devs,

I've purchased and used openaudible since version 2. So far as I can tell the application is only able to spawn three instances of ffmpeg to convert audiobooks. 

I've looked in the preferences, and FAQ for something like a "max simultaneous conversions" or "maximum threads to use for conversion" or something like that. I can't find anything like that.

My request is to add some setting for a user to customize the use of 1 core up to the max cores for their system for simultaneous book conversions. My M1 Macbook Pro and Intel desktop PC have 8 threads each. I would love to be able to use all of them at the same time on each machine. 

Thank you for your consideration of this feature!

**Comment by openaudible** [comment]
Right now it’s a hidden feature. (Trying to keep UI simple.) 

Open prefs, click the show preferences folder. Edit the settings.json file and change the max concurrent conversion to 10 from 3.

But probably should make the default higher and max threads 20 instead of 10. 

Those new macs are fast!

**Reply by pwnusmaximus** [reply]
I didn’t even think to check the json config file. 

Thank you! It works like a charm
